examples/inlinebot.py

```diff
@@ -48,31 +50,29 @@ def escape_markdown(text):
 
 
 def inlinequery(bot, update):
+    """Handle the inline query."""
     query = update.inline_query.query
-    results = list()
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Caps",
-                                            input_message_content=InputTextMessageContent(
-                                                query.upper())))
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Bold",
-                                            input_message_content=InputTextMessageContent(
-                                                "*%s*" % escape_markdown(query),
-                                                parse_mode=ParseMode.MARKDOWN)))
-
-    results.append(InlineQueryResultArticle(id=uuid4(),
-                                            title="Italic",
-                                            input_message_content=InputTextMessageContent(
-                                                "_%s_" % escape_markdown(query),
-                                                parse_mode=ParseMode.MARKDOWN)))
+    results = [
+        InlineQueryResultArticle(id=uuid4(),
+                                 title="Caps",
+                                 input_message_content=InputTextMessageContent(
+                                    query.upper())),
+        InlineQueryResultArticle(id=uuid4(),
+                                 title="Bold",
+                                 input_message_content=InputTextMessageContent(
+                                    "*{}*".format(escape_markdown(query)),
+                                    parse_mode=ParseMode.MARKDOWN)),
+        InlineQueryResultArticle(id=uuid4(),
+                                 title="Italic",
+                                 input_message_content=InputTextMessageContent(
+                                     "_{}_".format(escape_markdown(query)),
+                                     parse_mode=ParseMode.MARKDOWN))]
```

Comment:
autopep8 can be of assistant here as well.

Reply:
autopep8 did this to that part:
![ss 2017-10-15 at 10 49 27](https://user-images.githubusercontent.com/18517178/31589082-6d1b1a12-b1fb-11e7-9a99-61509a8ed7d7.png)

This is how it looked before:
![ss 2017-10-15 at 10 49 36](https://user-images.githubusercontent.com/18517178/31589083-7514df64-b1fb-11e7-868f-b8f79eef2d17.png)

imho the way it was before is more readable